Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
 members = [
-  "bee-tmp"
+  "bee-common",
+  "bee-common-derive"
```

Comment:
I wonder if we should bundle them together like we did in the dev branch. Just thinking that it can get a bit too much in the future if we have other derive crates.

Reply:
whatever you think is better is fine for me

Reply:
I'd rather bundle them yes